specification/newrelic/NewRelic.Management/examples/Organizations_List_MinimumSet_Gen.json

```diff
@@ -10,11 +10,7 @@
   "responses": {
     "200": {
       "body": {
-        "value": [
-          {
-            "id": "ycdsgeiitvxcd"
-          }
-        ]
+        "value": []
       }
```

Comment:
Add nextlink here?

Reply:
nextLink is not required
so in case when it is null, we wont be sending it. so the above example is correct. Its a min set